ca_description/meshes/LICENSE_create_1.txt

```diff
@@ -0,0 +1,31 @@
+Software License Agreement (BSD License)
```

Comment:
I got this license from the header of the source files of the turtlebot driver, but it probably is better just to link in the Readme. I didn't notice that you had already done that for other files.